### DISCUSSION TITLE
Levitating 100% scaled huge object

### DISCUSSION THREAD

**Discussion by burungiu** [google/model-viewer#1910] [Q&A]
Hi everyone,
We are developing a project, where a building is going to be viewed in AR, using model-viewer.
We succeeded to upload the model with textures in the app, but, unfortunately we have a levitating issue. The model, which is a huge building, is levitating 1-2 meters above the floor, and this is very annoying....
But, when I scale it to 1%, it stays perfect on the floor.
Did anyone had the same issue? How can it be solved?
Thx in advance,

Project is developed with the latest React.JS library. The project could be reached at: http://spital.imipasadegorj.ro (please be patient, it may load in 30 seconds or more.

- react-js: latest (i think)
- model-viewer: latest

### Browser Affected
- [*] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [*] Safari

### OS
- [*] Android
- [*] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
Thanks, I can repro that. It looks like the scale is being applied after our offset, so the offset is not being scaled in accordance. However, rather than waiting for a fix, I would recommend you rescale your glTF model. It looks like your building is properly modeled in meters, however if the goal is to view it in AR, you probably want to be showing a scale model. Since we begin the presentation at true scale, you would do well to set the scale of your model to whatever display scale makes sense to fit in a person's room.

**Comment by burungiu** [comment]
Ok, thank you for fast response
